pages/newsroom/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement, useContext } from "react";
+import { GetStaticProps, InferGetStaticPropsType } from "next";
+import { TypeNews } from "../../generated/contentful";
+import { getAllNewsEntries } from "../../contentful/client";
+import { getHeadPageTitle } from "../../utils/meta";
+import Head from "next/head";
+import Footer from "../../components/Footer";
+import NewsGrid from "../../components/News/NewsGrid";
+import Style from "./styles.module.scss";
+import LayoutContext from "../../components/layout/LayoutContext";
+import { PageSettings } from "../../types/next";
+
+export const pageSettings: PageSettings = {
+  path: "/newsroom",
+  linkContent: <>Newsroom</>,
+};
+
+export const getStaticProps: GetStaticProps<{
+  news: TypeNews[];
+}> = async () => {
+  let news = await getAllNewsEntries();
+
+  return {
+    props: {
+      news: news,
+    },
+  };
+};
+
+const Index = ({
+  news,
+}: InferGetStaticPropsType<typeof getStaticProps>): ReactElement => {
+  const { setNavColor } = useContext(LayoutContext);
+
+  setNavColor("black");
+
+  return (
+    <>
+      <Head>
+        <title>{getHeadPageTitle(["Newsroom"])}</title>
+      </Head>
+      <div className={Style.newsroomContainer}>
```

Comment:
Check some of the other pages like Legal. I might have an appropriate container and page title component set here you can standardize towards.

Reply:
They all use the same thing.